src/Uri.php

```diff
@@ -690,7 +701,13 @@ private function validateState()
                 throw new \InvalidArgumentException('A relative URI must not have a path beginning with a segment containing a colon');
             }
         } elseif (isset($this->path[0]) && $this->path[0] !== '/') {
-            throw new \InvalidArgumentException('The path of a URI with an authority must start with a slash "/" or be empty');
+            @trigger_error(
```

Comment:
Most people have an error handler that transforms any error to exception. So if you don't mute it, you could still say its a BC break for them. This approach is also used in symfony.